README.md

```diff
@@ -44,11 +44,19 @@ data () {
       :formState="formState"
       :formData="formData"
       :onValueChanged="onValueChanged"
+      :options="options"
       @addOptionRequest="handleAddOptionRequest"
       >
     </form-component>
 </template>
 ```
+### Options
+
```

Comment:
should note the options property itself is optional and that defaults are used when no options are provided